docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -233,6 +233,12 @@ They are detailed below.
 *Note :* If both an annotation attribute and a MicroProfile Config property are defined for the same option 
 then the MicroProfile Config property value always takes precedence over the `@OpenIdAuthenticationDefinition` annotation value.
 
+*Note :* If you are using MicroProfile Config properties and the value contains a placeholder that should not be resolved by MicroProfile Config
+itself but by the Server (like `payara.security.openid.redirectURI=${baseURL}/Callback`) make sure the value is properly escaped as it otherwise results in `NoSuchElementException`/
```

Comment:
```suggestion
itself but by the Server (like `payara.security.openid.redirectURI=${baseURL}/Callback`) make sure the value is properly escaped. It will otherwise result in `NoSuchElementException`. Here's an example how such a value should be escaped:
```